Common/Data/UniverseSelection/UniverseSettings.cs

```diff
@@ -56,13 +61,15 @@ public class UniverseSettings
         /// <param name="fillForward">True to fill data forward, false otherwise</param>
         /// <param name="extendedMarketHours">True to allow exended market hours data, false otherwise</param>
         /// <param name="minimumTimeInUniverse">Defines the minimum amount of time a security must remain in the universe before being removed</param>
-        public UniverseSettings(Resolution resolution, decimal leverage, bool fillForward, bool extendedMarketHours, TimeSpan minimumTimeInUniverse)
+        /// <param name="dataNormalizationMode">Defines the how the data is normalized</param>
+        public UniverseSettings(Resolution resolution, decimal leverage, bool fillForward, bool extendedMarketHours, TimeSpan minimumTimeInUniverse, DataNormalizationMode dataNormalizationMode = DataNormalizationMode.Adjusted)
```

Comment:
Is this the same default value as it was before? If not; need to use current default to avoid breaking user code.

Reply:
This is a new field. I introduced it to be used in the History request in Universe Selection and it is not used anywhere else.